Sources/MapboxNavigation/Style.swift

```diff
@@ -580,3 +580,24 @@ public class MarkerView: UIView {
         StyleKitMarker.drawMarker(innerColor: innerColor, shadowColor: shadowColor, pinColor: pinColor, strokeColor: strokeColor)
     }
 }
+
+/// :nodoc:
+@objc(MBFeedbackStyleView)
```

Comment:
As in https://github.com/mapbox/mapbox-navigation-ios/pull/3760#discussion_r813305601 try to minimize creation of public entities for the styling. I wonder if `UIAppearance.appearanceWhenContainedIn(_:)` can help here.

Reply:
I see. Thanks for the suggestion, I'll try to reduce the public exposed entities.